### PR TITLE
as_assistant column boolean, activeAdmin form updated, query updated

### LINKED ISSUE #28: Add as_assistant column to movies
- [x] Create a migration to add `as_assistant` columns to movies (set to `false` by default)
- [x] Change the query in `movies#index` controller in order to push movies as assistant at the end

### PR DESCRIPTION
**Pull request by benjyarel**
fixes #28 .

Schema updated , Active Admin have a checkbox for update/new . the movie index is order by as_assistant : true at the end of the list.

I'm waiting for this merge before working on #27  to skip conflicts

### CODE REVIEW

File: app/controllers/movies_controller.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class MoviesController < ApplicationController
   def index
-    @movies = Movie.order(year: :desc, priority: :asc, title: :asc).page(params[:page])
+    @movies = Movie.order(as_assistant: :asc, year: :desc, priority: :asc, title: :asc).page(params[:page])
```
Comment (grmnlrt) [inline app/controllers/movies_controller.rb]: Does this work? All film where `as_assistant` equal `true` are push at the end?

Comment (benjyarel) [reply app/controllers/movies_controller.rb]: Yes, because as_assistant: :asc is the first parameter of . order().

**Comment by benjyarel** [comment]
Like this ? 

Capture d’écran 2020-01-22 à 15 27 21

**Comment by benjyarel** [comment]
fixes #27 

I added published column and the scope on this branch too

File: app/controllers/movies_controller.rb (hunk 1 above)
Comment (grmnlrt) [reply app/controllers/movies_controller.rb]: OK I wasn't sure that `false` values would be at the beginning 👍

**Comment by grmnlrt** [comment]
> @grmnlrt Like this ?
> 
> Capture d’écran 2020-01-22 à 15 27 21

LGTM 👍 
Maybe just remove the italic. And can you add this label on the mobile version too?
When it's done you can merge 👌

**Pull request merged**